grpc/handlers.go

```diff
@@ -0,0 +1,65 @@
+package grpc
+
+import (
+	"fmt"
+	"github.com/NethermindEth/juno/db"
+	"github.com/NethermindEth/juno/grpc/gen"
+	"github.com/NethermindEth/juno/utils"
+)
+
+type handlers struct {
+	db db.DB
+}
+
+func (h handlers) Tx(server gen.KV_TxServer) error {
+	dbTx := h.db.NewTransaction(false)
+	tx := newTx(dbTx)
+	defer tx.cleanup()
```

Comment:
As a rule, we don't defer functions that can error.

See `db.CloseAndWrapOnError` and please use that to explicitly call cleanup on return statements.

Reply:
Rewrote it with `CloseAndWrap...` function.